### DISCUSSION TITLE
Naming styles

### DISCUSSION THREAD

**Discussion by HEAVYPOLY** [pocketbase/pocketbase#3254] [Polls]
Hi all, I've been using camelCase so far but read that sqlite is case insensitive.  Is camelCase still ok?

**Comment by ganigeorgiev** [comment]
I don't think there is anything wrong with using camelCase for collection and field names.
You are free to choose whatever style you want, just make sure that it is consistent.

I personally prefer camelCase because it is easier to type and that is what I use in my code (aka. both local client/server variables and JSON API responses follow the same convention).

**Reply by HEAVYPOLY** [reply]
Glad to hear, I'll stick with camelCase then.  It is nice to match JS names.

**Reply by VittorioParagallo** [reply]
I've been using till now snake_case but i'm starting to think that camelCase is better in the case of pocketbase, because even the parameter become more readable like, for example using indirect expansions. Ex student_card_via_card_code is more readable as studentCard_via_cardCode
